CMakeLists.txt

```diff
@@ -19,6 +19,11 @@ endif()
 
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# The OSX evergreen builder requires -stdlib=libc++ for cstdint to be defined
+if(CMAKE_OSX)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libc++")
```

Comment:
similarly, I think it would be better to do this from the evergreen script.  Logically, it feels like evergreen should know about cmake but cmake shouldn't necessarily know about evergreen.